bookstore/bookstore.py

```diff
@@ -1,29 +1,81 @@
+"""
+from hints below.
 def create_bookstore(name):
-    pass
+    return {
+        'name': name,
+        'last_author_id': 0,
+        'last_book_id': 0,
+        'books': [],
+        'authors': []
+    }
+"""
+
+
+def create_bookstore(name):
+    bookstore = {
+        'name': name, 'authors': [], 'books': [], 'author_id': 0, 'book_id': 0}
+    return bookstore
+
+
+#
+# From hints -
+# def add_author(bookstore, name, nationality):
+#    bookstore['last_author_id'] += 1
+#    author = {
+#        'name': name,
+#        'nationality': nationality,
+#        'id': bookstore['last_author_id']
+#    }
+#    bookstore['authors'].append(author)
+#    return author
+#
+
```

Comment:
Good job. There is a lot of comments from hints that make it slightly confusing where it almost looks like you have two def create_bookstore functions. I like the addition of the id fields.